### PR TITLE
Refactor path.join

### PR DESCRIPTION
**Pull request by victoriaxyz**

### CODE REVIEW

**Review by BeardOfDan — APPROVED** [review]

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -131,7 +131,7 @@ Thanks for contributing!
 
 This is just to help you organize your process
 
-- [ ] Did I cut my work branch off of master (don't cut new branches from existing feature brances)?
+- [ ] Did I cut my work branch off of master (don't cut new branches from existing feature branches)?
```
Comment (BeardOfDan) [inline CONTRIBUTING.md:134]: Good catch

**Review by BeardOfDan — APPROVED** [review]

File: server.js [hunk 2]
```diff
@@ -9,10 +9,10 @@ const bodyParser = require('body-parser');
 // This line sets the environment variables, since we are on our local machines
 // Therefore, in production (or whenever we are hosted on an actual server), this line can be removed
 //   along with the .env file
-require('dotenv').config();
-
-const db = require('./db/index');
+require('dotenv') // same as const dotenv = require('dotenv');
+.config(); // we just want to call .config, not save
```
Comment (philcho) [inline server.js:13]: I like the commenting. Maybe indent ".config();" so we know it's chained to the previous line.

File: server.js [hunk 3]
```diff
@@ -37,24 +37,24 @@ app.use(webpackDevMiddleware(compiler, {
...
-  res.sendFile(path.join(__dirname + '/dist/template.html'));
+  res.sendFile(path.join(__dirname, 'dist', 'template.html'));
 });
 
 // Capture GET requests for *.bundle.js and *.bundle.js.map files first. Otherwise, they'll get captured by the more-generic GET requests below (ex: '/profile/:id').
 app.get(/.+bundle.js.*/, (req, res, next) => {
   let filename = req.url.split('/')[2];
   // send the bundle.js or bundle.js.map file for that page
-  res.sendFile(path.join(__dirname + '/dist/' + filename));
+  res.sendFile(path.join(__dirname, 'dist', filename));
```
Comment (philcho) [inline server.js:47]: Nice fix, thanks.

File: server.js (hunk 2 above)
Comment (BeardOfDan) [reply server.js:13]: Oh yeah.. Phil is totally right

File: CONTRIBUTING.md (hunk 1 above)
Comment (victoriaxyz) [reply CONTRIBUTING.md:134]: Gotta catch 'em all.

File: server.js (hunk 2 above)
Comment (victoriaxyz) [reply server.js:13]: Makes sense to me.

**Pull request merged**